usr/share/rear/skel/default/etc/scripts/system-setup.d/40-start-udev-or-load-modules.sh

```diff
@@ -24,13 +24,11 @@ test "$systemd_version" || systemd_version=0
 if [[ $systemd_version -gt 190 ]] || [[ -s /etc/udev/rules.d/00-rear.rules ]] ; then
     # systemd-udevd case: systemd-udevd is started by systemd
     # Wait up to 10 seconds for systemd-udevd:
-    for countdown in 4 3 2 1 0 ; do
-        # The first sleep waits one second in any case so that systemd-udevd should be usually there
-        # when 'pidof' test for it so that usually there is no "Waiting for systemd-udevd" message:
-        sleep 1
+    for countdown in 10 9 8 7 6 5 4 3 2 1; do
+        # On modern systems systemd-udevd is already there at the very first iter
+        # So there may be no 'Waiting for systemd-udevd ($countdown) ... ' message at all 
```

Comment:
This change looks unrelated to what this pull request is about.
In general regarding changes for "modern systems" see
https://github.com/rear/rear/wiki/Coding-Style#maintain-backward-compatibility

Reply:
My concern is not about waiting longer.
Before it waited 4 times 2 seconds.
Now it waits 10 times one second.
My concern is about no longer waiting at least one second before
showing a "Waiting for systemd-udevd ($countdown) ... " message
(as far as I understand the code change on a first glance).
How does this change belong to what this pull request is about?

Reply:
It turns out that current PBA init is messed-up, particularly in udev part
* `sysinit-opalpba.service` depends on `systemd-udev-trigger.service`
* in addition `unlock-opal-disks` sources `40-start-udev-or-load-modules.sh`

which leads to multiple `udevadm trigger` calls
which may be dangerous being done in a row (w/o ensuring the ev-queue is empty/settled)

In particular this led to consistent hard-lockups of 990Pro nvme on one of my systems.

`sleep 1` sorts of masking the issue (by giving some time for already enqueued events to settle)

OpalPBA init needs some attention/rework, but that's far out of scope of this PR.
I reverted my previous change to `40-start-udev-or-load-modules.sh` to avoid unexpected side-effects

Reply:
In general regarding "udevadm" calls see 
https://github.com/rear/rear/issues/791
```
Waiting for udev and "kicking udev" are wrong (both miss the point)
```

I think it is the better workaround to simply sleep a bit
compared to interfering with what udev is doing.

Unfortunately we have a lot of "udevadm trigger"
and "udevadm settle" calls in current ReaR code usually
via the functions my_udevtrigger() and my_udevsettle()
in usr/share/rear/lib/linux-functions.sh
```
localhost:~/rear.github.master # find usr/sbin/rear usr/share/rear -type f | xargs grep -E 'my_udevsettle|my_udevtrigger|udevadm settle|udevadm trigger' | grep -v ': *#' | wc -l

43
```
where almost all of those cases need to be cleaned up properly
as time permits - which usually never happens because the
current state works somehow "good enough" in practice
(i.e. there are no noticable user issue reports)
and noone pays to get things implemented perfectly
as long as the current state works "good enough" for him.